trade-finance-manager-api/src/v1/deals/routes.js

```diff
@@ -78,7 +80,12 @@ dealsAuthRouter
 
 dealsAuthRouter
   .route('/deals/:dealId/amendments/:status?/:type?')
-  .get(validation.dealIdValidation, handleExpressValidatorResult, amendmentController.getAmendmentsByDealId);
+  .get(
+    validateUserHasAtLeastOneAllowedTeam([TEAM_IDS.PIM]),
+    validation.dealIdValidation,
```

Comment:
TODO: move this to new endpoint & test it

Reply:
Do we have a ticket for this

Reply:
Have done this as part of the PR now :)